src/openrct2/peep/Peep.cpp

```diff
@@ -3603,13 +3605,10 @@ static void peep_update_ride_prepare_for_state_9(rct_peep * peep)
     Ride * ride = get_ride(peep->current_ride);
 
     Guard::Assert(peep->current_ride_station < Util::CountOf(ride->exits), GUARD_LINE);
```

Comment:
This line is not changed in this commit, but is this assert correct?
Compare with, for example, the assert above on line 3097.
The scenario that I'm thinking of is coasters with exits on only some stations... but I have no idea whether this function is called in such cases.

Reply:
`Util::CountOf(ride->exits)` will always be `4`*, regardless of how many exits the ride actually has. 

\* It's the size of the array. We can change it by changing MAX_STATIONS, which in turn will change the size of `ride->exits`.